app/apps/server/orchestrator.js

```diff
@@ -122,6 +122,16 @@ settings.addGroup('General', function() {
 			hidden: false,
 		});
 
+		this.add('Apps_Framework_Development_Mode', false, {
```

Comment:
With this added, let's also change the logging so that it uses this setting instead of the `NODE_ENV`